src/docs/operator-guides/how-to-enable-pages/index.md

```diff
@@ -0,0 +1,38 @@
+---
+title: How to enable Pages in Console
+slug: how-to-enable-pages
+updated: 2023-02-14
+category: operator-guides
+ingress: Enable the Pages feature for your demo environment
+published: true
+---
+
+This article explains only how to enable Pages in your marketplace. If
+you want to learn how to use the feature, you can read the article about
+[editing Pages in Console](https://www.sharetribe.com/docs/operator-guides/how-to-edit-pages-in-console)
+
+## For marketplaces created after 2023-02-14 (PAGES RELEASE DATE)
+
+Log into the [Flex Console](https://flex-console.sharetribe.com/).
+Switch to the Demo environment. Choose the “Build” option from the top
+bar, then select “Content”. Go to the any of the default pages to start
+editing the corresponding page. When you make changes within the demo
+environment, you will be able to see changes right away. If you want to
+use the Pages feature in dev or production, you would need to do further
+setup with some development.
+
+## For marketplaces created before 2023-02-14 (PAGES RELEASE DATE)
+
+If you created your Flex marketplace before Pages release, you can try
+out Pages in your Demo environment by going to the home page of your
+Demo environment and clicking the "Refresh demo" button. You can then
+try editing them or creating new pages by navigating to "Build" and then
+"Content".
+
+Note that the above lets you test the Pages feature (and follow along
+our Pages documentation, including this
+[FAQ tutorial](https://www.sharetribe.com/docs/
```

Comment:
In the preview this markdown link is broken because of a space: https://flex-docs-git-pages-articles-sharetribe.vercel.app/operator-guides/how-to-enable-pages/